### ISSUE TITLE
Markdown table: double-quote ruins the output

### ISSUE THREAD

**Issue by PiotrGolonka** [doxygen/doxygen#8311]
Consider a markdown table with the cell that contains a single " (double-quote) character (as if you want to express the size of a screen in inches:

| SCREEN |SIZE |
|--------|-----|
| small  | 19" |
| large  | 27" |

After generating the LaTeX output one gets a table with a very strange content in the "SIZE":

```
small 19" \ilinebr </td> </tr> <tr class="markdownTableRowEven"> <td class="markdownTable←-
BodyNone"> large \ilinebr </td> <td class="markdownTableBodyNone"> 27"
```
When two double-quotes are used, this seems to be working fine.
Affects doxygen 1.9.0 (actually the "1.9.1" as in git's master branch as of today)

**Comment by albert-github** [comment]
The problem here is that the `"` is not terminated within the line and we have to use here 
- the command `\"` 
  See: https://www.doxygen.nl/manual/commands.html#cmdquot where it also states:
  >`\"`
  >
  > This command writes the " character to the output. This character has to be escaped in some cases, because it is used in pairs to indicate an unformatted text fragment.

or 
- the HTML-replacement: `&quot;` 
  See: https://www.doxygen.nl/manual/htmlcmds.html
  > `&quot;`     quotation mark = APL quote: " 

I think that we have here one of the unfortunate circumstances where the remark from the `\"` command holds.
That said on a lot of places even a single `"` might work, but it is a very tricky business.

When looking at version 1.8.16 it was working, but not with 1.8.17 and newer, looks like the the handling of `\ilinebr` in the current version in combination with the `"` is not 100% consistent.

**Comment by albert-github** [comment]
've just pushed a proposed patch, pull request #8315

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by PiotrGolonka** [comment]
Many thanks!

pt., 8 sty 2021 o 10:24 albert-github <notifications@github.com> napisał(a):

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.1.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
